From fb51a234caff7de1e3dda07ebb9744e37a95309e Thu Apr  1 21:56:04 2021
From: Simon Vogl <simon.vogl@gmx.net>
Date: Thu,  1 Apr 2021 21:56:04 UTC
Subject: [PATCH] Fix YouTube Search for QMPlay2.

This patch is required to fix QMPlay2's YouTube Search function by using an empty Cookie.

diff --git a/src/modules/Extensions/YouTube.cpp b/src/modules/Extensions/YouTube.cpp
index 78ee6755..c67c3434 100644
--- a/src/modules/Extensions/YouTube.cpp
+++ b/src/modules/Extensions/YouTube.cpp
@@ -501,7 +501,7 @@ void YouTube::search()
         if (lastTitle != title || sender() == searchE || sender() == searchB || qobject_cast<QAction *>(sender()))
         {
             m_currPage = 1;
-            searchReply = net.start(getYtUrl(title, m_sortByIdx));
+            searchReply = net.start(getYtUrl(title, m_sortByIdx), QByteArray(), "Cookie: \r\n");
         }
         else
         {
@@ -886,7 +886,7 @@ void YouTube::setSearchResults(const QJsonObject &jsonObj, bool isContinuation)
             {
                 tWI->setDisabled(true);
 
-                auto linkReply = net.start(url);
+                auto linkReply = net.start(url, QByteArray(), "Cookie: \r\n");
                 linkReply->setProperty("tWI", QVariant::fromValue((void *)tWI));
                 linkReplies += linkReply;
             }
